js/index.js

```diff
@@ -29,11 +21,69 @@ document.addEventListener("DOMContentLoaded", () => {
     "Project Management Systems (Jira, Asana, Wrike)",
     "Email Marketing (Klaviyo, Mailchimp)",
   ];
-  const skillsSection = document.getElementById('skills');
-  skillsList = skillsSection.querySelector('ul');
+  const skillsSection = document.getElementById("skills");
+  skillsList = skillsSection.querySelector("ul");
   for (let i = 0; i < skills.length; i++) {
     const skill = document.createElement("li");
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
   }
+
+
+
+  /////////////////////////////////////////
+  ///////////// MESSAGE FORM /////////////
+  ///////////////////////////////////////
+
+  // HIDE MESSAGES WHEN NONE//
+  document.getElementById("messages").style.display = "none";
+
+  // FORM //
+  const messageForm = document.getElementsByName("leave_message")[0];
+  messageForm.addEventListener("submit", (e) => {
+    e.preventDefault();
+
+    // USER INFO //
+    const usersName = e.target.usersName.value;
+    const usersEmail = e.target.usersEmail.value;
+    const usersMessage = e.target.usersMessage.value;
+    const newMessage = document.createElement("li");
+    newMessage.innerHTML = `<a href="mailto:${usersEmail}">${usersName} </a> wrote:<span> ${usersMessage}</span>`;
+
+    // MESSAGES //
+    const messageSection = document.getElementById("messages");
+    const messageList = messageSection.querySelector("ul");
+    // SHOW MSGS WHEN MSGS RECIEVED //
+    document.getElementById("messages").style.display = "";
+
+    // CREATE REMOVE BUTTON //
+    const removeButton = document.createElement("button");
+    removeButton.innerText = "remove";
+    removeButton.type = "button";
+
+    // REMOVE BUTTON //
+    removeButton.addEventListener("click", (e) => {
+      const entry = e.target.parentNode;
+      messageList.removeChild(entry);
+    });
```

Comment:
You commented:
> But for some reason, it doesn't disappear when I remove my message with the 'remove button'. It only seems to hide when I refresh the page. 

If you want more action associated with the remove button, you'd need to add it here, inside this callback.  Maybe you expect the code on lines 38-39 to run?:
```
  // HIDE MESSAGES WHEN NONE//
  document.getElementById("messages").style.display = "none";
```
but that's not inside a callback, so it will only run once, at page load.